recipe/meta.yaml

```
@@ -1,32 +1,36 @@
 {% set name = "httpx" %}
-{% set version = "0.23.0" %}
+{% set version = "0.26.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: f28eac771ec9eb4866d3fb4ab65abd42d38c424739e80c08d8d20570de60b0ef
-
+  sha256: 451b55c30d5185ea6b23c2c793abf9bb237d2a7dfb901ced6ff69ad37ec1dfaf
+  patches:
+    - patches/0001-remove-fancy-pypi-readme.patch
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [python_impl == 'pypy' and py==37]
-  skip: True  # [py<36 or s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<38 or s390x]
 
 requirements:
+  build:
+    - patch  # [unix]
+    - m2-patch  # [win]
   host:
     - pip
     - wheel
     - setuptools
     - python
+    - hatchling
   run:
+    - python
     - certifi
-    - httpcore >=0.15.0,<0.16.0
+    - httpcore ==1.*
+    - anyio
     - idna
-    - python
-    - rfc3986 >=1.3,<2
     - sniffio
```

```suggestion
    - sniffio
  run_constrained:
    - click >=8,<9
    - pygment >=2,<3
    - rich >=10,<14
    - h2 >=3,<5
    - socksio >=1,<2
```
From https://github.com/encode/httpx/blob/0.26.0/pyproject.toml#L39